get_box_mesh.py

```diff
@@ -0,0 +1,20 @@
+def get_box_mesh(dim,a,b,n, t=None, periodic=None):
```

Comment:
Let's remove this file and `get_box_mesh_v2.py`, and just keep the one in `mirgecom/simutil.py`.